aspnet/mvc/views/partial/sample/PartialViewsSample.sln

```diff
@@ -0,0 +1,32 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.25123.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{CD26E891-253A-464E-A354-A8759E7E7881}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{4DCCA7D8-6EAC-4717-89D2-5550DB50E8C9}"
+	ProjectSection(SolutionItems) = preProject
+		global.json = global.json
+	EndProjectSection
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "PartialViewsSample", "src\PartialViewsSample\PartialViewsSample.xproj", "{DA8E46F0-30E8-4C24-B81E-E315F93B4758}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{DA8E46F0-30E8-4C24-B81E-E315F93B4758}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{DA8E46F0-30E8-4C24-B81E-E315F93B4758}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{DA8E46F0-30E8-4C24-B81E-E315F93B4758}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{DA8E46F0-30E8-4C24-B81E-E315F93B4758}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(NestedProjects) = preSolution
+		{DA8E46F0-30E8-4C24-B81E-E315F93B4758} = {CD26E891-253A-464E-A354-A8759E7E7881}
+	EndGlobalSection
+EndGlobal
```

Comment:
@danroth27  can't we just add all those files to .gitignore - There must be some way to over-ride .gitignore in the rare case you want to include them. Steve makes a good point, it's extra work to exclude them. I do it with a script, but that might not work for everyone.